src/components/voter.jsx

```diff
@@ -0,0 +1,74 @@
+import { patchArticleVote } from "../utils/api-calls";
+import up from "../assets/up.png";
+import down from "../assets/down.png";
+import { useContext } from "react";
+import { ArticlesContext } from "../contexts/articles-context";
+
+export default function Voter({ article_id }) {
+  const { setArticles } = useContext(ArticlesContext);
+
+  const upVote = (article_id) => {
```

Comment:
None of these votes are optimistically rendering. It would also be great if the user would be given some feedback if their vote couldn't be registered!